app/helpers/ugc_form_helper.rb

```diff
@@ -6,4 +6,8 @@ module UGCFormHelper
   def date_format_fallback_attributes
     { pattern: '\d{4}-\d{2}-\d{2}', placeholder: 'YYYY-MM-DD' }
   end
+
+  def aasm_events_for_select(events)
+    events.map(&:name)
```

Comment:
might be an idea to look up a label/value for each here because I think 'wipe' will be misleading.